libpod/container.go

```diff
@@ -360,7 +361,6 @@ func (c *Container) RuntimeName() string {
 // This file will only be present after Init() is called to create the container
 // in runc
 func (c *Container) LogPath() string {
-	// TODO store this in state and allow overriding
 	return c.logPath()
```

Comment:
I think this needs to be changed to return LogPath as logPath only returns the filepath.Join() result now

Reply:
Actually I think we use logPath in a few other places

Reply:
Probably best to make logPath return c.config.LogPath and move the filepath.Join into NewContainer